op-acceptor/nat.go

```diff
@@ -443,6 +445,14 @@ func formatDuration(d time.Duration) string {
 	return fmt.Sprintf("%.1fs", d.Seconds())
 }
 
+// getErrorMessage returns the error message if test failed, empty string otherwise
+func getErrorMessage(status types.TestStatus, err error) string {
+	if status == types.TestStatusFail && err != nil {
+		return err.Error()
+	}
```

Comment:
The tiniest of nitpicks, just because I am a physicist by education and love haskell-like type safety: technically there is a case when `status == types.TestStatusFail` but `err == nil` in which this will return an empty error string - I personally would return some kinda default `TestStatusFail` message

Reply:
Added in https://github.com/ethereum-optimism/infra/pull/257